scripts/k8s_deploy_kubeflow.sh

```diff
@@ -19,8 +19,8 @@ export KFCTL="${KFCTL:-${CONFIG_DIR}/kfctl}"
 export KUBEFLOW_DEL_SCRIPT="${KF_DIR}/deepops-delete-kubeflow.sh"
 
 # Download URLs and versions # XXX: kfctl introcuded a version mismatch, this is naming only
-export KFCTL_FILE=kfctl_v1.0.2-0-ga476281_linux.tar.gz # https://github.com/kubeflow/kfctl/releases/tag/v1.0.2
-export KFCTL_URL="https://github.com/kubeflow/kfctl/releases/download/v1.0.2/${KFCTL_FILE}"
+export KFCTL_FILE=kfctl_v1.1-rc.1-0-g3e61b81_linux.tar.gz # https://github.com/kubeflow/kfctl/releases/tag/v1.1-rc.1
```

Comment:
v1.1.0 is out - can we upgrade to that instead of the RC? [kf releases](https://github.com/kubeflow/kfctl/releases)

Reply:
Done.